### PR TITLE
US888096: Use docker image version management plugin

### PR DESCRIPTION
**Pull request by kusumaghoshdastidar**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=888096

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -99,13 +100,49 @@
...
+                            <tag>4.0.0</tag>
+                            <digest>sha256:03c2b7fe78f3db1831a492ca341266eca7efb92f1f4968939308b9d038aba97c</digest>
+                        </image>
+                        <image>
+                            <repository>${dockerHubPublic}/cloudesire/webdav</repository>
+                            <tag>latest</tag>
+                        </image>
+                        <image>
+                            <repository>${dockerHubPublic}/library/postgres</repository>
+                            <tag>14.12-alpine3.19</tag>
```
Comment (dermot-hardy) [inline pom.xml:133]: How will this ever be updated given the blanket ban on alpine images?

https://github.houston.softwaregrp.net/sepg/sepg-dependencies-command/blob/16133a89d7430d1508afae32823ad3d1d2629470/src/main/docker/command/ignoreDockerVersions.yaml#L28-L29

**Review by dermot-hardy — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (kusumaghoshdastidar) [reply pom.xml:133]: Should we pick _bookworm_ or _bullseye_ instead? 
Remove "*alpine" from ignore versions?
Configure ignoreVersions per repository and not have "*alpine" ignored for postgres?

Comment (dermot-hardy) [reply pom.xml:133]: Why are we wanting to ban alpine images?

Comment (kusumaghoshdastidar) [reply pom.xml:133]: I added it for rabbitmq where were specifically looking for `management` tags and I wanted to include other tags in the ignore list.

Comment (dermot-hardy) [reply pom.xml:133]: > I added it for rabbitmq...

Do we have a way in the ignore rules file to say that the rule applies just for a particular image or set of images?  If not then we'll need to raise a ticket and add that.

Comment (kusumaghoshdastidar) [reply pom.xml:133]: No, will implement applying rules just for a particular image or set of images as part of [916303](https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=916303)

**Pull request merged**